features/admins/admin_invitations.feature

```diff
@@ -398,6 +398,16 @@ Feature: Admin Actions to Manage Invitations
     Then I should see "oldman@ds9.com"
       And I should see "Invitation was successfully sent."
 
+  Scenario: Sent to field is a link for redeemed invitations
+    Given the user "creator" exists and is activated
+      And the user "invitee" exists and is activated
+      And an invitation created by "creator" and used by "invitee"
+      And I am logged in as a super admin
+    When I go to creator's manage invitations page
+    When I follow the first invitation token url
+    Then I should see "Sent to"
+      And I should see a link "invitee" within "dl"
```

Comment:
This is the username link. It would be nice to check for the email link, since that's what is modified here, and also follow it and check that you end up on the filtered invitation search. Looking at the factory, I think the email is `default@email.com`?